Source/Scene/LabelCollection.js

```diff
@@ -103,6 +125,44 @@ define([
         var glyphIndex;
         var textIndex;
 
+        var showBackground = label._showBackground;
+        var backgroundBillboard = label._backgroundBillboard;
+        if (!showBackground) {
```

Comment:
Possibly, I'm just mirroring what is already done with `_spareBillboards` for glyphs.  We can't have the spare billboards (for glyphs or backgrounds) shared between separate label collections, which is probably why the spares are owned by the collection instance.